tests/unit/core/test_api_errors.py

```
@@ -0,0 +1,661 @@
+
+
+@pytest.mark.asyncio
+async def test_successful_json_stream_is_forwarded_incrementally():
+    app = _build_app()
+    first_chunk_received = asyncio.Event()
+    second_chunk_received = asyncio.Event()
+    body_chunks = []
+    request_received = False
+    disconnect = asyncio.Event()
+
+    async def receive():
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `receive`

(ANN202)